common/gtk-2.0/main.rc

```diff
@@ -31,7 +31,7 @@ style "default" {
   GtkRange::stepper-size = 0
 
   GtkScrollbar::activate-slider = 1
-  GtkScrollbar::has-backward-stepper = 0  
+  GtkScrollbar::has-backward-stepper = 0
```

Comment:
Next time it would be better to split whitespace fixes in a single comment. I usually safe the file (my editor autoremoves them), commit the changes and then do my fixes. Its okay for this minor change, but for larger PRs its harder to review. You possibly already knew that, I am lazy too sometimes :D